lib/util/miq_apache/miq_apache.rb

```diff
@@ -125,7 +125,7 @@ def self.status(full = true)
     end
 
     def self.version
-       MiqUtil.runcmd("rpm -qa --queryformat '%{VERSION}' httpd")
+      MiqUtil.runcmd("rpm -qa --queryformat '%{VERSION}' #{MiqApache.package_name}")
```

Comment:
@Fryguy Yes, in the past I've thought that it would be nice to extract this to LinuxAdmin, but agree that it's out of scope for this PR.